subsys/bluetooth/services/hids.c

```diff
@@ -327,7 +327,7 @@ static ssize_t hids_outp_rep_write(struct bt_conn *conn,
 		    .data = rep_data,
 		    .size = rep->size,
 		};
-		rep->handler(&report, conn);
+		rep->handler(&report, conn, true);
```

Comment:
Please make it consistent and add a callback to the:
```hids_outp_rep_read```

Reply:
done